api/models/Detainees.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+var LinkingModels = require('sails-linking-models');
+var ModelHelpers = require('../lib/ModelHelpers');
+
+const model = {
+  schema: true,
+  autoCreatedAt: true,
+  autoUpdatedAt: true,
+  autoPK: false,
+  attributes: {
+    person_id: {
```

Comment:
And the same on Event model? Or can Event have something more meaningful given the context of the content, i.e,
```js
attributes: {
    detainee: {
        model: 'detainees'
    }
}
```

Rather then what we currently have, which is:
```js
attributes: {
    person_id: {
        model: 'detainees'
    }
}
```
The first example makes more sense to me because the model is not an id (string), it is an entity, and so should have a meaningful name?